pkg/steampipeconfig/load_config.go

```
@@ -49,6 +50,24 @@ func LoadConnectionConfig() (*SteampipeConfig, *modconfig.ErrorAndWarnings) {
 
 func ensureDefaultConfigFile(configFolder string) error {
 	defaultConfigFile := filepath.Join(configFolder, defaultConfigFileName)
+	defaultConfigSampleFile := filepath.Join(configFolder, defaultConfigSampleFileName)
+	// if default.spc exists, and if the content of the default.spc and the default.spc.sample
+	// file are the same, remove the default.spc file.
+	if _, err := os.Stat(defaultConfigFile); err == nil {
+		if equal, _ := utils.AreFilesEqual(defaultConfigFile, defaultConfigSampleFile); equal {
```

don't ignore the error. INFO log it at least